<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconfigurable Intelligent Surface - RIS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="RIS_tab">
       <attribute name="title">
        <string>RIS</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="config_tab">
       <attribute name="title">
        <string>Konfiguration</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>141</width>
          <height>111</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="QGB_SerielConfig">
           <property name="title">
            <string>RIS serielle Verbindung</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="PB_connect">
              <property name="text">
               <string>verbinden</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PB_disconnect">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>trennen</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="PB_update_RIS">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>update RIS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="PB_reset">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>117</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>RIS Reset</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="text">
    <string>Ã–ffnen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
